linera-chain/src/manager.rs

```diff
@@ -849,11 +832,27 @@ impl ChainManagerInfo {
 
     /// Returns whether the `identity` is allowed to propose a block in `round`.
     /// This is dependent on the type of round and whether `identity` is a validator or (super)owner.
-    pub fn can_propose(&self, identity: &AccountOwner, round: Round) -> bool {
+    pub fn can_propose(
+        &self,
+        identity: &AccountOwner,
+        round: Round,
+        seed: u64,
+        info: &ChainManagerInfo,
```

Comment:
Is that a different one than `self`??

Reply:
🙈

Reply:
Fixed in [Don't pass in the manager (it's self)](https://github.com/linera-io/linera-protocol/pull/4695/commits/af92770f6383a606acdbde037587ae0e16dec9c2).